services/director-v2/src/simcore_service_director_v2/api/routes/services.py

```diff
@@ -63,3 +70,29 @@ async def get_service_versioned(
     forward_request: Response = Depends(forward_to_director_v0),
 ):
     return forward_request
+
+
+@router.post(
+    "/{service_key:path}/{service_version}/dynamic-sidecar:require",
```

Comment:
this entrypoint name is very weird. Why is this needed at all?

Reply:
To see if a dynamic service can be part of the project networks, it is required to check if it is started via dynamic-sidecar.

Better naming suggestions are welcomed.